src/ts/view/view.ts

```diff
@@ -334,8 +301,7 @@ const viewSearchResults = () => {
 };
 
 
-// Remove duplicates - but should be fixed in cypher
-const formatNames = (array: []) => [...new Set(array)].map(x => `"${x}"`).join(', ');
+const formatNames = (array: []) => [...new Set(array)].map(x => `“${x}”`).join(', ');
```

Comment:
```suggestion
const formatNames = (array: []) => [...new Set(array)].map(x => `"${x}"`).join(', ');
```

Reply:
Aren't the curly quotes the best ones to use for user-visible text?